files/default/neutron-ha-tool.py

```diff
@@ -944,10 +949,30 @@ def pick(self):
 
 
 class NullRouterFilter(object):
+    def load(self):
+        pass
+
     def filter_routers(self, router_id_list):
         return router_id_list
 
 
+class ListFileBasedRouterFilter(object):
+    def __init__(self, router_list_file):
+        self.router_list_file = router_list_file
+        self.router_list = []
+
+    def load(self):
+        self.router_list = []
+        with open(self.router_list_file, 'rb') as fh_router_list:
```

Comment:
any reason for read binary? This would cause problems if we ever move to py3 as it would return a `bytes` object instead of a `str`